minitwit-api/sim/sim-facade.go

```diff
@@ -2,15 +2,24 @@ package sim
 
 import (
 	"encoding/json"
+	"minitwit-api/db"
+	"minitwit-api/logger"
 	"net/http"
-	"os"
+	"strconv"
 )
 
+var lg = logger.InitializeLogger()
+
 func UpdateLatest(r *http.Request) {
 	r.ParseForm()
 	latest := r.Form.Get("latest")
 	if latest != "" {
-		_ = os.WriteFile("./latest_processed_sim_action_id.txt", []byte((latest)), 0644)
+		db, err := db.GetDb()
+		if err != nil {
+			lg.Fatal("Failed to get DB", err)
+		}
+		val, _ := strconv.Atoi(latest)
```

Comment:
Does the code need to read the latest value from the file the first time that we run this on the server? So that we don't loose the history.

Reply:
I considered it, but from what I can tell they update the value with every request, but only read the value every few hours.
The odds of it being an issue is close to non-existing.